lib/streams/json-parse-stream.js

```diff
@@ -0,0 +1,433 @@
+var ZTransform = require('../transform');
+var util = require('util');
+
+/*global Buffer*/
+// Named constants with unique integer values
+var C = {};
```

Comment:
Don't include the library here.
Use the npm module [jsonparse](https://www.npmjs.com/package/jsonparse) and wrap it in a ZStream, which overrides jsonparse's `onError` and `onValue` functions.